### ISSUE TITLE
frost-core: update for spec version 6&7

### ISSUE THREAD

**Issue by dconnolly** [ZcashFoundation/frost#86]
Depends on #60 being done

https://github.com/cfrg/draft-irtf-cfrg-frost/compare/draft-irtf-cfrg-frost-05...draft-irtf-cfrg-frost-07

Linking to the git compare as the spec changelog hasn't been updated fully

draft-06:

- [ ] Make verification a per-ciphersuite functionality: #91 
- [ ] Use per-signer values of rho to mitigate protocol malleability: #92

**Comment by conradoplg** [comment]
Already done in the linked PRs
